state/state.go

```diff
@@ -142,3 +144,36 @@ func (b *BabelState) Listen(updates chan Transition) error {
 	active = false
 	return nil
 }
+
+func (b *BabelState) ListenHistory() error {
```

Comment:
I need something like this, because it want reuse this library put just without any Websocket / channel foo for liveupdates.

I like to read the "History" from time to time with:
https://github.com/Vivena/babelweb2/blob/0a9e624a4e6e463ab92fe12fea3162b0b0e9a1b4/state/state.go#L73

So if you do not like the name `ListeHistory` please make a alternative suggestion for it.

Reply:
It is not about the name. If you change the code for your internal experiments that's cool, but how is this related to us?

Reply:
It is not related to babelweb2, but why should your `state` and `parser` package not be useable by other programms?

Reply:
Sure it might be useable, and it's great that someone find it interesting. This is an open software and you very welcome to use and change it as you like, but babelweb is not a library or something. Why include code that is not related to our purpose?